discord/guild.py

```diff
@@ -676,6 +676,19 @@ def categories(self) -> list[CategoryChannel]:
         r.sort(key=lambda c: (c.position or -1, c.id))
         return r
 
+    @property
+    def activity_feed_enabled(self) -> bool:
+        """Returns ``True`` if the guild has the activity feed enabled."""
+        return "ACTIVITY_FEED_DISABLED_BY_USER" not in self.features
+
+    @property
+    def invites_disabled(self) -> bool:
+        """Returns ``True`` if the guild has invites disabled.
+
+        This corresponds to the invites being paused under the server's Security Actions
+        """
+        return "INVITES_DISABLED" in self.features
+
```

Comment:
We don't have attributes for other features, besides Guild.invites_disabled exists already.